ehsm_kms_service/router.js

```diff
@@ -68,6 +70,20 @@ const find_cmk_by_keyid = async (appid, keyid, res, DB) => {
   return keyBlob
 }
 
+const GetRouter = async (p) => {
+  const { req, res } = p
+  const action = req.query.Action
+  switch (action) {
+    case common_apis.GetVersion:
+      napi_res = napi_result(action, res, [])

```

Comment:
so the Getversion is a httpGet request?  if so, please update the commit-msg title.

Reply:
It fixed.